apps/frontend/src/app/PageHome/index.tsx

```diff
@@ -21,6 +21,7 @@ import QuickLinksCard from './QuickLinksCard'
 import ResinCard from './ResinCard'
 import TeamCard from './TeamCard'
 import VidGuideCard from './VidGuideCard'
+import package_json from 'package.json'
 
 export default function PageHome() {
   // TODO: translations
```

Comment:
Really random, but could you remove this comment. Looks like we translated everything anyways.